src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,360 @@
+<?php namespace Ushahidi\Console\Command;
+
+/**
+ * Ushahidi Obfuscate Data console command
+ *  - This overwrites specific data within the current database
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Illuminate\Console\Command;
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Core\Entity\ConfigRepository;
+use Ushahidi\Core\Entity\UserRepository;
+use Ushahidi\App\Multisite;
+use Ohanzee\DB;
+use DB as LaravelDB;
+use Faker;
+use Database;
+
+class ObfuscateData extends Command
+{
+    private $postRepository;
+    private $contactRepository;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'db:obfuscatedata';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'db:obfuscatedata
+        {--use-faker : Uses the slower Faker library to generate realistic data }';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Obfuscates selected PII and removes various config keys from database';
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->db = service('kohana.db');
+        $this->contactRepository = service('repository.contact');
+        $this->postRepository = service('repository.post');
+        $this->configRepository = service('repository.config');
+        $this->userRepository = service('repository.user');
+    }
+
+    public function handle()
+    {
+        if ($this->isThisAMultisiteInstall()) {
+            if (!getenv('HOST') || strlen(getenv('HOST')) < 1) {
+                $this->alert("ERROR: A host must be specified for a multisite deployment.");
+                return;
+            }
+        }
+
+        if ($this->isTestEnvironment() || $this->isStagingEnvironment()) {
```

Comment:
We might want to replace this with a similar method to whats used in deployments:import

ie.
```
            if (!$this->getLaravel->environment($this->allowedEnvironments) && $this->options('overwrite-force')) {
                $this->warn('Overwriting existing deployments in a production environment');
            }
```